selenium/Tests/UploadFileTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using OpenQA.Selenium;
+using System;
+
+namespace SeleniumTests.Tests
+{
+    class UploadFileTests
+    {
+        public static readonly TestCase UploadSHACLFile =
+            new TestCase(
+            "SHACL files can be uploaded",
+        (driver, log) =>
+        {
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            //Log in
+            driver.Login();
+            String currentPath = System.IO.Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName;
```

Comment:
Why not insert a `using System.IO;` directive at the top of the file and drop the qualifiers here? `Directory.GetParent` is just as clear as `System.IO.Directory.GetParent` and a lot shorter.